lib/hammer_cli/apipie/option_builder.rb

```diff
@@ -66,7 +66,7 @@ def option_opts(param)
       opts[:required] = true if (param.required? and require_options?)
       if param.expected_type == :array || param.validator =~ /Array/i
```

Comment:
Use the same style here, either convert to a string like with boolean or use a symbol comparison below.